Formula/pdmenu.rb

```diff
@@ -0,0 +1,34 @@
+class Pdmenu < Formula
+  desc "Full screen menuing system for Unix"
+  homepage "https://joeyh.name/code/pdmenu/"
+  url "http://sources.buildroot.net/pdmenu/pdmenu-1.3.6.tar.gz"
+  sha256 "dedd2a4a5719278b5e49041161990c2f20b5909818837542aaca01819f2c14eb"
+  license "GPL-2.0-only"
+
+  head "git://git.joeyh.name/pdmenu"
+
+  depends_on "gettext" => :build
+  depends_on "s-lang" => :build
+
+  def install
+    ENV.append "CC", "-I#{HOMEBREW_PREFIX}/include"
+    ENV.append "SLANG_H_LOC", "#{HOMEBREW_PREFIX}/include/slang.h"
+    ENV.append "SLANG_LIB_LOC", "#{HOMEBREW_PREFIX}/lib"
+    ENV.append "INSTALL", "#{HOMEBREW_PREFIX}/bin/ginstall"
+
+    # should be fixed in the upcoming 1.3.7
+    system "sed", "-i", "-c", "s?^LIBS.*$?LIBS           = @LIBS@ @INTLLIBS@ $(EFENCE)?", "autoconf/makeinfo.in"
+
+    # we cannot use "/usr/share/locale"
+    system "sed", "-i", "-c", "s?^LOCALEDIR.*$?LOCALEDIR      = $(INSTALL_PREFIX)/#{prefix}/locale?", "Makefile"
```

Comment:
There's no need to shell out to `sed` for this. You can use [`inreplace`](https://rubydoc.brew.sh/Formula.html#inreplace-instance_method).

Also, we don't actually accept new formulae that need patching, so we'll probably wait until 1.3.7.

Reply:
The problem with pdmenu is the lack of source tarball. The official git repo doesn't offer the possibility to download an archive, and the "official" tarball is in Debian pool, in version 1.3.4. And this version requires many patches to compile with a modern gcc and OS.

The second sed is because there is a "/usr/share/" in the makefile, used in make install

Reply:
Please use `inreplace` instead of `sed`.